packages/react-dev-utils/formatBuildError.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const get = require('lodash/get');
+const chalk = require('chalk');
+
+module.exports = function formatBuildError(err) {
+  const message = get(err, 'message');
+
+  // Add more helpful message for UglifyJs error
+  if (typeof message === 'string' && message.indexOf('from UglifyJs') !== -1) {
+    try {
+      console.log(
+        'Failed to minify the code from \n\n',
+        chalk.yellow(
+          /Unexpected token:(.+)\[(.+)\]\[(.+)\]/.exec(err.stack)[2]
```

Comment:
This could not match or be undefined (in the future if the text format changes); we probably shouldn't assume it's always going to be OK (we don't want this method throwing an error).